### ISSUE TITLE
`semantic-line-breaks` conflict with `mkdocs` flavor

### ISSUE THREAD

**Issue by tahv** [rvben/rumdl#484]
### rumdl version

0.1.36

### Installation method

uv

### Bug description

The following diagnostic show up on list items when `semantic-line-breaks` is used with `mkdocs` flavor:

> Paragraph should use semantic line breaks (1 sentences) [MD013]

The warning only seem to show up if the list item have 3 lines or more.

### Steps to reproduce

`test.md` :

```md
# Test

- Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nullam consectetur tortor eu ipsum fringilla venenatis.
- Fusce sollicitudin nibh nec magna molestie porta.
  Quisque scelerisque libero non nisi tristique,
  ac dignissim lacus sollicitudin.
```

### Expected behavior

No diagnostic.

### Actual behavior

`rumdl check`:

```console
$ rumdl check test.md
repro.md:6:1: [MD013] Paragraph should use semantic line breaks (1 sentences) [*]

Issues: Found 1 issues in 1 file (29ms)
Run `rumdl fmt` to automatically fix 1 of the 1 issues
```

`rumdl fmt`:

```console
$ rumdl fmt test.md
repro.md:6:1: [MD013] Paragraph should use semantic line breaks (1 sentences) [fixed]

Fixed: Fixed 1/1 issues in 1 file (41ms)

$ cat test.md
# Repro

- Lorem ipsum dolor sit amet, consectetur adipiscing elit.
  Nullam consectetur tortor eu ipsum fringilla venenatis.
- Fusce sollicitudin nibh nec magna molestie porta.
Quisque scelerisque libero non nisi tristique, ac dignissim lacus sollicitudin.

```

### Configuration

```toml
[tool.rumdl]
flavor = "mkdocs"
MD013.reflow = true
MD013.reflow-mode = "semantic-line-breaks"
```

### Operating system

Windows

**Comment by rvben** [comment]
Fixed in [v0.1.37](https://github.com/rvben/rumdl/releases/tag/v0.1.37). MD013 now correctly detects MkDocs 2-space list continuation indents in `semantic-line-breaks` mode. Thanks for reporting this @tahv!
